src/Platforms/AbstractPlatform.php

```diff
@@ -4638,6 +4642,16 @@ public function columnsEqual(Column $column1, Column $column2): bool
         return $column1->getType() === $column2->getType();
     }
 
+    /**
+     * Whether the database data type matches that expected for the doctrine type.
+     *
+     * To be implemented in specific platforms as necessary.
+     */
+    public function columnDeclarationsMatch(Column $column1, ?Column $column2 = null): bool
```

Comment:
Apart from tests: When is this method ever called with one parameter or with the second parameter set to `null`?

Reply:
Method signature amended to require two parameters.